backend/core/src/shared/enums/video/video-filters-data.config.ts

```diff
@@ -0,0 +1,74 @@
+import { StreamingStatus } from '@prisma/client';
+import { DurationFilterId, DateFilterId, TypeFilterId, SortByFilterId } from 'shared/build';
+
+type DurationOption = {
+  lte: number | undefined;
+  gte: number | undefined;
+};
+
+type DateOption = Date | undefined;
+
+type SortByOption = Array<[string, string | undefined]>;
+
+const matchVideoFilterDuration: Record<DurationFilterId, DurationOption> = {
+  [DurationFilterId.UNDER_4_MINUTES]: {
+    lte: 240,
+    gte: undefined,
+  },
+  [DurationFilterId.ANY]: {
+    lte: undefined,
+    gte: undefined,
+  },
+  [DurationFilterId.FROM_4_TO_20_MINUTES]: {
+    lte: 1200,
+    gte: 240,
+  },
+  [DurationFilterId.OVER_20_MINUTES]: {
+    lte: undefined,
+    gte: 1200,
+  },
+};
+
+const matchTimeValuesInMilliseconds = {
+  [DateFilterId.LAST_HOUR]: 3600000,
+  [DateFilterId.THIS_WEEK]: 604800000,
+  [DateFilterId.THIS_MONTH]: 2592000000,
+  [DateFilterId.THIS_YEAR]: 31557600000,
+};
```

Comment:
Try to write it down like 1000 * 60 * 60 and etc.
It'll be easier to read and check manualy.